README.md

```diff
@@ -1,59 +1,36 @@
 
-Repositories that fall under the responsibility of the TSC and CommComm:
+This repository is formal efforts of the Node.js project that are overseen by the [TSC](https://github.com/nodejs/TSC) regarding the less technical aspects of the project.
+
+It is also used to discuss:
 
-- This repo
 - [Getting Started](https://github.com/nodejs/getting-started)
 - The private [Moderation](https://github.com/nodejs/moderation) repo (Collaborators-only access)
 
 ## Travel Fund
 
-Each member of the Node.js organization can ask for funding to travel and spread knowledge about and support the Javascript ecosystem and the Foundation.
+Each member of the Node.js organization can ask for funding to travel and spread knowledge about and support the JavaScript ecosystem and the Foundation.
 
 The management of Travel Fund for collaborators is managed by the [OpenJS
 Foundation Cross-Project-Council](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/project-resources/MEMBER_TRAVEL_FUND.md).
 
 ## Collaborator Summit
 
-Every year, Node.js collaborators meet up in one or more Collaborator
-Summit. If you are interested in joining or helping with the
-organization, the management of Collaborators Summit is managed by the [OpenJS
-Foundation Summit team](https://github.com/openjs-foundation/summit).
+Every year, Node.js collaborators meet up in one or more Collaborator Summit. If you are interested in joining or helping with the organization, the management of Collaborators Summit is managed by the [OpenJS Foundation Summit team](https://github.com/openjs-foundation/summit).
 
 ## Governance and Current Members
 
-The Admin repo is overseen by the [TSC](https://github.com/nodejs/TSC) and [CommComm](https://github.com/nodejs/community-committee) as collaborative groups. Any action requiring consensus and voting will abide by the following process.
-
-Members of the TSC and CommComm shall operate under Lazy Consensus as a collaborative unit in the Admin repo hereby referenced as the *Admin members*. Members shall establish appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the development process.
-
-For all votes, the vote must pass in both TSC and CommComm or else the motion fails. If an individual happens to be in both committees, then they would end up with two votes.
+The Admin repo is overseen by the [TSC](https://github.com/nodejs/TSC). Any action requiring consensus and voting will abide by the TSC process for that.
 
 ### Contacts for assistance
-- [@bnb](https://github.com/bnb) - **Tierney Cyren**, CommComm Chair
-- [@joesepi](https://github.com/joesepi) - **Joe Sepi**, CommComm OpenJS Representative
 - [@mhdawson](https://github.com/mhdawson) - **Michael Dawson**, TSC Chair
+- [@mcollina](https://github.com/mcollina) - **Matteo Collina**, TSC Vice Chair
+- [@joesepi](https://github.com/joesepi) - **Joe Sepi**, CommComm OpenJS Representative
```

Comment:
@joesepi isn't the CommComm representative - there is no CommComm :P

Reply:
Joe and I are the two Node.js reps on the CPC. Both reps are appointed by TSC. It used to be that one was TSC and one was CommComm.

Neither of us need to be listed here. We don't manage the admin repo. TSC does. So just listing the TSC chair and vice chair seems good enough.

Reply:
```suggestion
```